src/SDK/OssClient.cs

```diff
@@ -46,6 +46,7 @@ public class OssClient
         private SigningStyleService signingStyleService;
         private DataRetentionSettingsService dataRetentionSettingsService;
         private JsonSerializerSettings jsonSerializerSettings;
+        private DeclineReasonsService declineReasonsService;
```

Comment:
This service is not required because AccountConfigService will be used. Please delete it.